Fix subprocess result bytes not being decoded in SunOS/Solaris related tools.

The fix was offered upstream:
https://github.com/SCons/scons/pull/3453

--- scons-3.1.1/engine/SCons/Tool/suncxx.py
+++ scons-3.1.1/engine/SCons/Tool/suncxx.py
@@ -74,7 +74,7 @@ def get_package_info(package_name, pkgin
         except EnvironmentError:
             pass
         else:
-            pkginfo_contents = p.communicate()[0]
+            pkginfo_contents = p.communicate()[0].decode()
             version_re = re.compile(r'^ *VERSION:\s*(.*)$', re.M)
             version_match = version_re.search(pkginfo_contents)
             if version_match:
@@ -88,7 +88,7 @@ def get_package_info(package_name, pkgin
             except EnvironmentError:
                 pass
             else:
-                pkgchk_contents = p.communicate()[0]
+                pkgchk_contents = p.communicate()[0].decode()
                 pathname_re = re.compile(r'^Pathname:\s*(.*/bin/CC)$', re.M)
                 pathname_match = pathname_re.search(pkgchk_contents)
                 if pathname_match:
